tools/test/examples/examples.json

```diff
@@ -37,12 +37,15 @@
       "mbed": [
           "https://developer.mbed.org/teams/mbed-os-examples/code/mbed-os-example-mesh-minimal"
       ],
-      "features" : ["IPV6"],
-      "targets" : [],
+      "features" : [],
+        "targets" : ["DISCO_F469NI", "DISCO_F746NG", "K64F", "K66F",
```

Comment:
My editor seems to think that the entire file is misaligned. I'll fix this instance shortly